coalib/collecting/Collectors.py

```diff
@@ -214,7 +214,7 @@ def collect_all_bears_from_sections(sections, log_printer):
     """
     Collect all kinds of bears from bear directories given in the sections.
 
-    :param bear_dirs:   directory name or list of such that can contain bears
+    :param sections:    directory name or list of such that can contain bears
```

Comment:
what I'm saying here is that `sections` is no `directory name or list of such that can contain bears`!